components/src/main/res/layout/component_editable_list_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <uk.gov.hmrc.components.atom.heading.Heading5
+            android:id="@+id/title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/hmrc_spacing_24"
+            android:focusable="true"
+            android:gravity="start"
+            tools:text="Title" />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/list_item"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            tools:listitem="@layout/editable_list_item" />
+
+        <uk.gov.hmrc.components.atom.button.SecondaryButton
+            android:id="@+id/secondary_button"
+            style="@style/Text.Body"
```

Comment:
If we change this to use the IconButton we shouldn't need to define a style here, as we want the standard usage of our existing IconButton atom.